solution/build.sh

```diff
@@ -86,7 +86,9 @@ function copy_yamls()
 
     mkdir -p ${crd_dir} ${role_dir}
 
-    kustomize build "${zenko_operator_repo}/config/artesca-solution/crd?ref=$(zenko_operator_tag)" > ${crd_dir}/zenko_crd.yaml
+    kustomize build "${zenko_operator_repo}/config/crd/bases/zenko.io_zenkoconfigurationoverlays.yaml?ref=$(zenko_operator_tag)" > ${crd_dir}/zenkoconfigurationoverlays_crd.yaml
```

Comment:
minor: using `-o` should allow each file to be generated separately
```suggestion
    kustomize build "${zenko_operator_repo}/config/artesca-solution/crd?ref=$(zenko_operator_tag)" -o ${crd_dir}
    rename '.yaml' '_crd.yaml' ${crd_dir}/*

```